lib/content-services/src/lib/dialogs/dialog/dialog.component.html

```diff
@@ -0,0 +1,63 @@
+<div
+    data-automation-id="adf-dialog-container"
+    class="adf-dialog-container {{dialogSize}}"
+>
+    <div
+        mat-dialog-title
+        data-automation-id="adf-dialog-header"
+        class="adf-dialog-header"
+        [ngClass]="{'adf-centered-header': data.headerIcon}"
+    >
+        <div class="adf-dialog-title-container" [ngClass]="{'adf-centered-title': data.headerIcon}">
+            <mat-icon *ngIf="data.headerIcon" class="adf-dialog-header-icon">{{data.headerIcon}}</mat-icon>
+            <h2 class="adf-dialog-title">{{data.title}}</h2>
+        </div>
+        <button
+            *ngIf="!isCloseButtonHidden"
+            mat-icon-button
+            mat-dialog-close
+            data-automation-id="adf-dialog-close-button"
+        >
+            <mat-icon>close</mat-icon>
+        </button>
+    </div>
+
+    <div
+        [ngTemplateOutlet]="data.descriptionTemplate"
+        [ngClass]="{'adf-description': data.description || data.descriptionTemplate }"
+    >
+        <ng-container>{{data.description}}</ng-container>
+    </div>
+
+    <mat-dialog-content class="adf-dialog-content">
+        <ng-container [ngTemplateOutlet]="data.contentTemplate"></ng-container>
+    </mat-dialog-content>
+
+    <mat-dialog-actions class="adf-dialog-actions" [ngClass]="{'adf-additional-actions': data.actionsTemplate}">
+        <div>
+            <ng-container [ngTemplateOutlet]="data.actionsTemplate"></ng-container>
+        </div>
+        <div>
+            <button
+                *ngIf="!isCancelButtonHidden"
+                mat-button
+                mat-dialog-close
+                adf-auto-focus
+                data-automation-id="adf-dialog-actions-cancel"
+            >
+                {{cancelButtonTitle | translate}}
+            </button>
+
+            <button
+                mat-button
+                mat-flat-button
```

Comment:
`mat-button` can be removed if we have  `mat-flat-button`?

Reply:
yes, you're right, updated